app/lib/actions/katello/product/content_create.rb

```diff
@@ -52,9 +52,9 @@ def finalize
             new_content.vendor = ::Katello::Provider::CUSTOM
           end
 
-          #custom product content is always enabled by default
+          # custom product content is always disabled by default
           ::Katello::ProductContent.where(product: root.product, content: content).first_or_create do |pc|
-            pc.enabled = true
+            pc.enabled = false
```

Comment:
..and this is what sets it in the Katello web UI / Repository Sets. We have to make sure we match Candlepin's value or bad things will happen.